schema-language/src/main/antlr4/Courier.g4

```diff
@@ -158,7 +158,7 @@ NULL_LITERAL: 'null';
 
 SCHEMADOC_COMMENT: '/**' .*? '*/';
 BLOCK_COMMENT: '/*' .*? '*/' -> skip;
-LINE_COMMENT: '//' ~['\r\n']* -> skip;
+LINE_COMMENT: '//' ~[\r\n]* -> skip;
```

Comment:
Yes, that's what we want, with high confidence.
A substring that starts with `//` and continues with any number of characters not `\r` or `\n` is a single-line (and non-doc) comment and thus should be skipped.